examples/step-40/step-40.cc

```diff
@@ -644,12 +650,6 @@ namespace Step40
           refine_grid();
 
         setup_system();
-
-        pcout << "   Number of active cells:       "
-              << triangulation.n_global_active_cells() << std::endl
-              << "   Number of degrees of freedom: " << dof_handler.n_dofs()
-              << std::endl;
-
         assemble_system();
         solve();
```

Comment:
Since I see the scope of the computing timer two lines below, would you mind moving that into the `output_results` section, just as we have the respective scopes inside the `assemble_system` and `solve` functions? That would further enhance readability.

Reply:
Already in #16862.